src/server/game/Entities/Creature/Creature.cpp

```diff
@@ -3727,7 +3728,7 @@ time_t Creature::GetLastLeashExtensionTime() const
     return *GetLastLeashExtensionTimePtr();
 }
 
-void Creature::UpdateLeashExtensionTime()
+void Creature::UpdateLeashExtensionTime() const
```

Comment:
Why make it const? We are modifying the object here, so it doesn't make sense.

Reply:
VS hint can do this.
Well, I removed it